### PR TITLE
fix: Allow only one order per attendee

### LINKED ISSUE #4901: Multiple Orders can be linked to an attendee.
**Describe the bug**

Right now we can multiple orders related to an attendee.

**To Reproduce**
1. Create two orders relating it to the same attendee.

**Expected behavior**

An attendee should have one and only one related order.

**Additional context**

Parent issue: #4852

### PR DESCRIPTION
**Pull request by srv-twry**
Fixes #4901 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Right now we can multiple orders related to an attendee.

#### Changes proposed in this pull request:
Raise ConflictException when creating order for a TicketHolder already having a related Order.

### CODE REVIEW

**Comment by srv-twry** [comment]
# Order successfully created
order_created_success

# Conflict on trying the same one again
now_it_says_conflict

**Comment by srv-twry** [comment]
> Isn't this extremely inefficient?
Instead of a trying this query and raising an error if it was not completed, why not just use a conditional to check if the order already exists, with a simple if

@Kreijstal I am unable to figure out how we can achieve that using a simple `if`. Can you like explain a bit more. 

Note that I had made a mistake earlier and now I rectified it. So please check my code again and if you still think we can achieve it using an `if`, then please explain. Thanks

File: app/api/orders.py [hunk 1]
```diff
@@ -50,6 +51,19 @@ def before_create_object(self, data, view_kwargs):
         :param view_kwargs:
         :return:
         """
+        for ticket_holder in data['ticket_holders']:
+            """
+                Ensuring that the attendee exists and has doesn't have an associated order.
+            """
```
[the remark this answers is not in the document]
Comment (srv-twry) [reply app/api/orders.py]: Umm, what should I do about this ? I wanted to make things clear hence added the comment.

**Comment by maxlorenz** [comment]
LGTM

**Pull request merged**